src/snakemake/workflow.py

```
@@ -177,6 +178,7 @@ class Workflow(WorkflowExecutorInterface):
     _rundir = str(Path.cwd().absolute())
     _workdir_handler: Optional[WorkdirHandler] = field(init=False, default=None)
     injected_conda_envs: List = field(default_factory=list)
+    start_time: datetime = datetime.now()
```

Ruff (0.16.1)

[warning] Do not perform function call `datetime.now` in dataclass defaults

(RUF009)